_data/mentors.yml

```diff
@@ -591,3 +591,148 @@
   network:
     - linkedin: https://www.linkedin.com/in/kirthikka-devi-v
     - Medium: https://medium.com/@rithik18
+
+- name: Mona Ahluwalia
+  disabled: false
+  sort: 100
+  hours: 3
+  position:  Talent Director, Chapter 2 
+  type: ad-hoc
+  index: 19
+  location: London, UK
+  bio: |
+    I'm a Recruitment professional and start-up scaling specialist. I can help mentees with the soft-skills development to really make impact in their role and in the workplace.
+  image: assets/images/mentors/mona_ahluwalia.jpg
+  languages: English
+  skills:
+    experience: 5-7 Years
+    years: 7
+    mentee: I can help with all soft-skills related coaching
+    areas:
+      - 
```

Comment:
I guess it is missing areas and languages no?

Reply:
Mona has no programming languages listed on the original website, and only one bullet point for areas which I've put into focus (I don't think it allowed me to put an empty focus).
![Screenshot 2024-05-01 144550](https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/assets/11363288/aa6545d8-4f4e-4963-8ed8-fd8f926fffaf)
![Screenshot 2024-05-01 144557](https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/assets/11363288/097d3506-a3a9-41a1-a91c-c377146b6c4d)